roles/ups/templates/prometheus_rule.yml.j2

```
@@ -0,0 +1,144 @@
+# Monitor Service (Metrics)
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    monitoring-key: middleware
+    prometheus: application-monitoring
+    role: alert-rules
+  name: unifiedpush
+spec:
+  selector:
+    matchLabels:
+      app: unifiedpush
+  groups:
+    - name: general.rules
+      rules:
+      - alert: UnifiedpushDown
+        expr: absent(kube_pod_container_status_running{namespace="{{ ups_namespace }}",container="ups"}==1)
+        for: 5m
+        labels:
+          severity: critical
+        annotations:
+          description: "The UnifiedPush Pod Server has been down for more than 5 minutes. "
+          summary: "The UnifiedPush Pod Server is down. For more information see on the UnifiedPush at https://github.com/aerogear/aerogear-unifiedpush-server"
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
+      - alert: UnifiedPushConsoleDown
+        expr: absent(kube_endpoint_address_available{endpoint="example-unifiedpushserver-unifiedpush"} == 1)
+        for: 5m
+        labels:
+          severity: critical
+        annotations:
+          description: "The UnifiedPush admin console has been down for more than 5 minutes. "
+          summary: "The UnifiedPush admin console endpoint has been unavailable for more that 5 minutes. For more information see on the UnifiedPush  at https://github.com/aerogear/UnifiedPush "
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
+      - alert: UnifiedPushDatabaseDown
+        expr: absent(kube_pod_container_status_running{namespace="{{ ups_namespace }}",container="postgresql"}==1)
+        for: 5m
+        labels: 
+          severity: critical
+        annotations:
+          description: "The UnifiedPush Database pod has been down for more than 5 minutes"
+          summary: "The UnifiedPush Database is down. For more information see on the UnifiedPush at https://github.com/aerogear/aerogear-unifiedpush-serve"
+          sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-push.adoc" #todo
+      - alert: UnifiedPushPodCPUHigh # todo check if it will work after expse the metrcis
```

```suggestion
      - alert: UnifiedPushPodCPUHigh # todo check if it will work after expose the metrics
```

Have we checked if this works? If so, should we remove the todo? If not, will adding the alert here cause any undesirable behaviour, like constantly alerting?

Same comment for all of the other ones that have a similar "todo" comment.